src/proxy-ui-api/frontend/src/views/GenerateCertificateSignRequest/WizardPageCsrDetails.vue

```diff
@@ -0,0 +1,207 @@
+          <v-select
+            :items="localMembersIds"
+            item-text="id"
+            item-value="id"
+            class="form-input"
+            v-model="client"
+          ></v-select>
+        </ValidationProvider>
+      </div>
+
+      <div class="row-wrap">
+        <div class="label">
+          {{$t('csr.certificationService')}}
+          <helpIcon :text="$t('csr.helpCertificationService')" />
+        </div>
+
+        <ValidationProvider name="crs.certService" rules="required" v-slot="{ errors }">
+          <v-select
+            :items="filteredServiceList"
+            item-text="name"
+            item-value="name"
+            class="form-input"
+            v-model="certificationService"
+          ></v-select>
+        </ValidationProvider>
+      </div>
+
+      <div class="row-wrap">
+        <div class="label">
+          {{$t('csr.csrFormat')}}
+          <helpIcon :text="$t('csr.helpCsrFormat')" />
+        </div>
+
+        <ValidationProvider name="crs.crsFormat" rules="required" v-slot="{ errors }">
+          <v-select
+            :items="csrFormatList"
+            name="crs.crsFormat"
+            class="form-input"
+            v-model="csrFormat"
+          ></v-select>
+        </ValidationProvider>
+      </div>
+
+      <div class="button-footer">
+        <large-button outlined @click="cancel">{{$t('action.cancel')}}</large-button>
+        <large-button :disabled="invalid" @click="done">{{$t('action.continue')}}</large-button>
+      </div>
+    </ValidationObserver>
+  </div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import { mapGetters } from 'vuex';
+import { ValidationProvider, ValidationObserver } from 'vee-validate';
+import HelpIcon from '@/components/ui/HelpIcon.vue';
+import LargeButton from '@/components/ui/LargeButton.vue';
+import SubViewTitle from '@/components/ui/SubViewTitle.vue';
+import { Key, Token } from '@/types';
+import { UsageTypes, CsrFormatTypes } from '@/global';
+import * as api from '@/util/api';
+
+export default Vue.extend({
+  components: {
+    HelpIcon,
+    LargeButton,
+    SubViewTitle,
+    ValidationObserver,
+    ValidationProvider,
+  },
+  props: {},
+  data() {
+    return {
+      usageTypes: UsageTypes,
+      usageList: Object.values(UsageTypes),
+      csrFormatList: Object.values(CsrFormatTypes),
+    };
+  },
+  computed: {
+    ...mapGetters([
+      'localMembersIds',
+      'filteredServiceList',
+      'isUsageReadOnly',
+    ]),
+
+    usage: {
+      get() {
+        return this.$store.getters.usage;
+      },
+      set(value) {
+        this.$store.commit('storeUsage', value);
+      },
+    },
+    csrFormat: {
+      get() {
+        return this.$store.getters.csrFormat;
+      },
+      set(value) {
+        this.$store.commit('storeCsrFormat', value);
+      },
+    },
+    client: {
+      get() {
+        return this.$store.getters.csrClient;
+      },
+      set(value) {
```

Comment:
This could be typed (as string I understood correctly).

This comment applies to other parameters without a type in this file

Reply:
Fixed